example_1/package.xml

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>ros2_control_demo_hardware</name>
+  <name>ros2_control_demo_example_1</name>
   <version>0.0.0</version>
-  <description>Demo package of `ros2_control` Hardware. This package provides example on how to define hardware of your own robot for ROS2 control. The package is used with `ros2_control_demo_bringup` from package `rrbot_description` and `diffbot_description`.</description>
+  <description>Demo package of `ros2_control` hardware for RRbot.</description>
 
-  <maintainer email="denis@stogl.de">Denis Štogl</maintainer>
+  <maintainer email="denis.stogl@stoglrobotics.de">Dr.-Ing. Denis Štogl</maintainer>
+  <maintainer email="christoph.froehlich@ait.ac.at">Dr.techn. Christoph Froehlich</maintainer>
```

Comment:
I know I'm cherry picking but could we please drop the academic decorations from ROS package maintainer names?

Reply:
I'm fine with that ;)